### PR TITLE
Timestamp

### PR DESCRIPTION
**Pull request by chopj**
Added TimeStamp tag to kml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,12 @@ you define a mapping from the GeoJSON style to KML's.
 * `description`: the name of the property in each GeoJSON Feature that contains
   the feature's description
 
+**Timestamp:** KML can associate features with a moment in time via the `TimeStamp` tag.  GeoJSON doesn't
+have a comparable field, but a custom property can be mapped
+
+* `timestamp`: the name of the property in each GeoJSON Feature that contains 
+  a timestamp in XML Schmea Time (yyyy-mm-ddThh:mm:sszzzzzz)
```
Comment (tmcw) [inline README.md]: sp here, should be Schema

**Comment by tmcw** [comment]
:+1: will need a manual merge, I'll do that now.

**Comment by tmcw** [comment]
Merged in https://github.com/mapbox/tokml/pull/21 - thanks, and sorry for the delay!

**Comment by chopj** [comment]
Thanks for merging!

**Pull request merged**